### ISSUE TITLE
Feature - Add MH in the report

### ISSUE THREAD

**Issue by opimon** [alephium/mining-pool#23]
Hi,
Can you support  this strandard message for MH reporting ? (Bzminer)
Like this we can report the MH per miner. it would be great

alph_submitHashrate

example:
mining-pool_1     | 2022-01-20T10:51:49.736Z | error | Unknown stratum method from ::ffff:81.221.123.56:48960: alph_submitHashrate

**Comment by AcidBurnSB** [comment]
Agree, official miner useless for 10xx series GPU, BZMiner giving more Mh/s for all series, and almost x3 for 10xx series and more information, but this error for Hashrate... brrr...

**Comment by AcidBurnSB** [comment]
BzMiner from v7.2.0 fix this issue, need to close.

PS: error still available:(

**Comment by altinsoft** [comment]
Yes. error still available:(
